src/Parser.ts

```diff
@@ -45,35 +45,16 @@ export class _Parser {
    * Parse Loop
    */
   parse(tokens: Token[], top = true): string {
-    let out = '',
-      i,
-      j,
-      k,
-      l2,
-      l3,
-      row,
-      cell,
-      header,
-      body,
-      token,
-      ordered,
-      start,
-      loose,
-      itemBody,
-      item,
-      checked,
-      task,
-      checkbox,
-      ret;
+    let out = '';
 
-    const l = tokens.length;
-    for (i = 0; i < l; i++) {
-      token = tokens[i];
+    for (let i = 0; i < tokens.length; i++) {
```

Comment:
Is there any perf difference between storing `tokens.length` in a variable vs checking in here each time? I suspect we won't see much of a different if the length is small but it might add up for large arrays.

Reply:
There is no performance benefit from storing the variable first. https://stackoverflow.com/a/17995000/806777